### PR TITLE
Check if HTML is escaped properly before echoing

### LINKED ISSUE #407: Check if HTML is echoing safely
Check if HTML is echoing safely to prevent being flagged by WP tools.

### PR DESCRIPTION
**Pull request by alexmigf**
closes #407

### CODE REVIEW

**Comment by alexmigf** [comment]
@YordanSoares @dwalkerpriv @dpeyou could you please test this changes and let me know if you find any glitch in the PDFs or in the settings?

**Comment by dwalkerpriv** [comment]
@alexmigf seems fine for me. anything specific i should check?

**Comment by alexmigf** [comment]
@dwalkerpriv just to a overall check, printing every data as possible in the PDF and see it everything works as normal. Also try to save/update some settings.

**Review by DavidAnderson684 — APPROVED** [review]
@alexmigf Approved - but I think I failed to clarify: they're not threatening to review this plugin. It's only the ones that they closed that have to be reviewed. For the ones still open, they've only requested the change of name.

File: templates/Simple/invoice.php [hunk 1]
```diff
@@ -106,21 +106,21 @@
...
 		<?php foreach ( $this->get_order_items() as $item_id => $item ) : ?>
-			<tr class="<?php echo apply_filters( 'wpo_wcpdf_item_row_class', 'item-'.$item_id, $this->get_type(), $this->order, $item_id ); ?>">
+			<tr class="<?php echo apply_filters( 'wpo_wcpdf_item_row_class', 'item-'.$item_id, esc_attr( $this->get_type() ), $this->order, $item_id ); ?>">
 				<td class="product">
 					<?php $description_label = __( 'Description', 'woocommerce-pdf-invoices-packing-slips' ); // registering alternate label translation ?>
-					<span class="item-name"><?php echo $item['name']; ?></span>
+					<span class="item-name"><?php echo esc_html( $item['name'] ); ?></span>
 					<?php do_action( 'wpo_wcpdf_before_item_meta', $this->get_type(), $item, $this->order  ); ?>
-					<span class="item-meta"><?php echo $item['meta']; ?></span>
+					<span class="item-meta"><?php echo esc_html( $item['meta'] ); ?></span>
```
Comment (Terminator-Barbapapa) [inline templates/Simple/invoice.php:114]: Should be `wp_kses_post()`.  Item meta HTML is now escaped.

Schermafbeelding 2022-10-20 om 10 34 37

File: templates/Simple/packing-slip.php [hunk 2]
```diff
@@ -93,20 +93,20 @@
...
 		<?php foreach ( $this->get_order_items() as $item_id => $item ) : ?>
-			<tr class="<?php echo apply_filters( 'wpo_wcpdf_item_row_class', 'item-'.$item_id, $this->get_type(), $this->order, $item_id ); ?>">
+			<tr class="<?php echo apply_filters( 'wpo_wcpdf_item_row_class', 'item-'.$item_id, esc_attr( $this->get_type() ), $this->order, $item_id ); ?>">
 				<td class="product">
 					<?php $description_label = __( 'Description', 'woocommerce-pdf-invoices-packing-slips' ); // registering alternate label translation ?>
-					<span class="item-name"><?php echo $item['name']; ?></span>
+					<span class="item-name"><?php echo esc_html( $item['name'] ); ?></span>
 					<?php do_action( 'wpo_wcpdf_before_item_meta', $this->get_type(), $item, $this->order  ); ?>
-					<span class="item-meta"><?php echo $item['meta']; ?></span>
+					<span class="item-meta"><?php echo esc_html( $item['meta'] ); ?></span>
```
Comment (Terminator-Barbapapa) [inline templates/Simple/packing-slip.php:101]: Should be `wp_kses_post()`.

**Review by Terminator-Barbapapa — CHANGES_REQUESTED** [review]

**Review by YordanSoares — APPROVED** [review]
## Invoice:
image

## Packing Slip:
image

**Pull request merged**